src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -0,0 +1,1024 @@
+            vmax_l1_m2 = exp(vslope_l1_m2 * t_soi24_degC_col(c,j) + vint_l1_m2) * vmod_l1_m2
+            vmax_l2_m1 = exp(vslope_l2_m1 * t_soi24_degC_col(c,j) + vint_l2_m1) * vmod_l2_m1
+            vmax_l2_m2 = exp(vslope_l2_m2 * t_soi24_degC_col(c,j) + vint_l2_m2) * vmod_l2_m2
+            vmax_s3_m1 = exp(vslope_s3_m1 * t_soi24_degC_col(c,j) + vint_s3_m1) * vmod_l2_m1
+            vmax_s3_m2 = exp(vslope_s3_m2 * t_soi24_degC_col(c,j) + vint_s3_m2) * vmod_l2_m2
+
+            ! km are time-dependent terms
+            km_l1_m1 = exp(kslope_l1_m1 * t_soi24_degC_col(c,j) + kint_l1_m1) * kmod_l1_m1
+            km_l1_m2 = exp(kslope_l1_m2 * t_soi24_degC_col(c,j) + kint_l1_m2) * kmod_l1_m2
+            km_l2_m1 = exp(kslope_l2_m1 * t_soi24_degC_col(c,j) + kint_l2_m1) * kmod_l2_m1
+            km_l2_m2 = exp(kslope_l2_m2 * t_soi24_degC_col(c,j) + kint_l2_m2) * kmod_l2_m2
+            km_s3_m1 = exp(kslope_s3_m1 * t_soi24_degC_col(c,j) + kint_s3_m1) * kmod_s3_m1 / p_scalar(c,j)
+            km_s3_m2 = exp(kslope_s3_m2 * t_soi24_degC_col(c,j) + kint_s3_m2) * kmod_s3_m2 / p_scalar(c,j)
+
+            ! Product of w_scalar * depth_scalar * o_scalar
+            w_d_o_scalars = w_scalar(c,j) * depth_scalar(c,j) * o_scalar(c,j)
+
+            ! decomp_k used in SoilBiogeochemPotentialMod.F90
+            ! TODO Defined correctly here?
+            ! TODO Combine multiple terms of same donor pools...
+            ! TODO Where to calculate m1_pool and m2_pool?
+            ! TODO When to introduce litter/soil biomass to the calculation?
+            ! TODO Excluded spinup terms for now. See cwd below how used.
+            decomp_k(c,j,i_met_lit) = (vmax_l1_m1 * m1_pool(c,j) / &
+                                       (km_l1_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_met_lit) = (vmax_l1_m2 * m2_pool(c,j) / &
+                                       (km_l1_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
+
+            decomp_k(c,j,i_str_lit) = (vmax_l2_m1 * m1_pool(c,j) / &
+                                       (km_l2_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_str_lit) = (vmax_l2_m2 * m2_pool(c,j) / &
+                                       (km_l2_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
+
+            decomp_k(c,j,i_pas_som) = (vmax_s3_m1 * m1_pool(c,j) / &
+                                       (km_s3_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_pas_som) = (vmax_s3_m2 * m2_pool(c,j) / &
+                                       (km_s3_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
+
+            decomp_k(c,j,i_act_som) = k_s1_s3 * w_d_o_scalars
+            decomp_k(c,j,i_slo_som) = k_s2_s3 * w_d_o_scalars
```

Comment:
s1_s3 --> s3_s1
s2_s3 --> s2_s1
both of these should be controlled by microbial biomass:
the flux in the testbed is "oxidat":
[OXIDAT = (mimicspool%MICk(npt) * mimicsbiome%Vmax(npt,K2) * mimicspool%SOMc(npt)    &
                    / (mimicsbiome%KO(2)*mimicsbiome%Km(npt,K2) + mimicspool%SOMc(npt)))      &
                    + (mimicspool%MICr(npt) * mimicsbiome%Vmax(npt,R2) * mimicspool%SOMc(npt) &
                    / (mimicsbiome%KO(1)*mimicsbiome%Km(npt,R2) + mimicspool%SOMc(npt)))](https://github.com/wwieder/biogeochem_testbed_1.1/blob/52d460fd39720c3f7dc7b47265f20f4c6149796d/SOURCE_CODE/mimics_cycle_CN.f90#L897)

Reply:
oxidat shows two components going from s2 to s1, the sum of which is decomp_k.

Reply:
Initially I misunderstood from comment 1 above that both s2_s1 and s3_s1 were controlled by OXIDAT. From comment 2 and the testbed code I now realize that we were likely talking about s2_s1 only. And the decomp_k term for s3_s1 just equals desorption from what I can tell.